src/applications/disability-benefits/all-claims/pages/privateMedicalRecordsRelease.js

```diff
@@ -0,0 +1,151 @@
+import _ from '../../../../platform/utilities/data';
+import { merge } from 'lodash';
+import fullSchema526EZ from 'vets-json-schema/dist/21-526EZ-schema.json';
+import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
+import { uiSchema as addressUI } from '../../../../platform/forms/definitions/address';
+import {
+  disabilityNameTitle,
+  recordReleaseDescription,
+  limitedConsentTitle,
+  limitedConsentTextTitle,
+  limitedConsentDescription,
+} from '../content/privateMedicalRecordsRelease';
+
+import { validateZIP } from '../validations';
+import PrivateProviderTreatmentView from '../components/PrivateProviderTreatmentView';
+import { validateDate } from 'us-forms-system/lib/js/validation';
+
+const { address } = fullSchema526EZ.definitions;
+
+export const uiSchema = {
+  'ui:description': recordReleaseDescription,
+  'ui:title': disabilityNameTitle,
+  'view:limitedConsent': {
+    'ui:title': limitedConsentTitle,
+  },
+  limitedConsent: {
+    'ui:title': limitedConsentTextTitle,
+    'ui:options': {
+      expandUnder: 'view:limitedConsent',
+      expandUnderCondition: true,
+    },
+    'ui:required': (formData, index) =>
+      _.get(`disabilities.${index}.view:limitedConsent`, formData),
+  },
+  'view:privateRecordsChoiceHelp': {
+    'ui:description': limitedConsentDescription,
+  },
+  providerFacility: {
+    'ui:options': {
+      itemName: 'Provider',
+      viewField: PrivateProviderTreatmentView,
+      hideTitle: true,
+    },
+    items: {
+      providerFacilityName: {
+        'ui:title': 'Name of private provider or hospital',
+      },
+      'ui:validations': [validateDate],
+      treatmentDateRange: dateRangeUI(
+        'Approximate date of first treatment',
+        'Approximate date of last treatment',
+        'End of treatment must be after start of treatment',
+      ),
+      providerFacilityAddress: merge(addressUI('', false), {
+        street2: {
+          'ui:title': 'Street 2',
+        },
+        postalCode: {
+          'ui:title': 'Postal Code',
+          'ui:options': {
+            widgetClassNames: 'usa-input-medium',
+          },
+          'ui:validations': [
```

Comment:
Using a pattern on the schema would remove the need to run a custom validator. Here's one for the 526 MVP's primary address zip:
`pattern: '^\\d{5}(?:([-\\s]?)\\d{4})?$'`

Reply:
@markgreenburg - Actually, after taking a look at it, we use the address definition in the schema, which uses the baseAddressDef, which has that exact pattern on it. This means that this validation portion shouldn't be necessary lol.

Reply:
🎉